samples/algorithms/QuantumHelloWorld.qs

```diff
@@ -0,0 +1,24 @@
+/// # Quantum Hello World!
+///
+/// This Q# code generates a random bit by setting a qubit in a superposition of the
+/// computational basis states |0〉 and |1〉, and returning the measurement result.
+namespace QuantumHelloWorld {
+    // Import the `Message` function and the `H` operation used in the program.
+    open Microsoft.Quantum.Intrinsic;
+
+    @EntryPoint()
+    operation RandomBit() : Result {
+        Message("Hello world!");
```

Comment:
In the spirit of commenting on everything, it might be worth an explanatory comment here that `Message` is how Q# programs print to stdout.

Reply:
I suggested to Cesar to make comments more succinct to keep Hello World as lean as possible. We plan to make this sample as default code on quantum.microsoft.com. The goal here is to show how easy it is to start with Q#, so we can skip commenting on code that is intuitive for non-quantum developers.